File: python/src/tests/t_clock_id.py
#!/usr/bin/python3
#
# $Id:$
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

r = atrshmlog.attach()

id = atrshmlog.get_clock_id()

print('clock id : ' + str(id) + ' : ')

oldid = atrshmlog.set_clock_id(2)

print('clock id : ' + str(oldid) + ' : ')

id = atrshmlog.get_clock_id()

print('clock id : ' + str(id) + ' : ')

oldid = atrshmlog.set_clock_id(1)

print('clock id : ' + str(oldid) + ' : ')

id = atrshmlog.get_clock_id()

print('clock id : ' + str(id) + ' : ')



print (' ')

exit(0);

# end of test

File: perl/src/tests/t_clock_id.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$id = Atrshmlog::get_clock_id();

print "clock id : $id : \n";

$oldid = Atrshmlog::set_clock_id(2);

print "clock id : $oldid : \n";

$id = Atrshmlog::get_clock_id();

print "clock id : $id : \n";

$oldid = Atrshmlog::set_clock_id(1);

print "clock id : $oldid : \n";

$id = Atrshmlog::get_clock_id();

print "clock id : $id : \n";


print " \n";

exit (0);

# end of main
